### DISCUSSION TITLE
Show summary of values present across all results

### DISCUSSION THREAD

**Discussion by vurtual1** [microsoft-search/pnp-modern-search#2820] [Q&A]
Hi,

I'm trying to work out if its possible to show values only once, a bit like groupings. I have a list containing Customer Name and Product, where Product could be say (widget, gadget, toy or bicycle). If I have a query of Customer Name = Contoso I get a result of:
Name       Product
====       =====
Contoso   Bicycle
Contoso   Bicycle
Contoso   Widget
Contoso   Widget
Contoso   Bicycle
Contoso   Widget
Contoso   Bicycle

Is it possible at all to format my results to just show:
Products

Bicycle
Widget

This is so I can see at a glance what product types the customer is buying, but not every instance of them. I've tried searching the discussion for groupings, instance, count etc but can't see anything relevant, apologies if I'm just not using the right search magic.

Thanks!

**Comment by wobba** [comment]
Refiners are aggregates, so using the refinement/filter results somehow might work. That or if someone adds support for collapsespec https://learn.microsoft.com/en-us/sharepoint/dev/general-development/customizing-search-results-in-sharepoint#collapse-similar-search-results-using-the-collapsespecification-property
